bc_obps/service/operation_service.py

```diff
@@ -606,7 +615,9 @@ def generate_boro_id(cls, user_guid: UUID, operation_id: UUID) -> Optional[BcObp
         operation.save(update_fields=['bc_obps_regulated_operation'])
         if operation.bc_obps_regulated_operation is None:
             raise Exception('Failed to create a BORO ID for the operation.')
-
+        send_registration_and_boro_id_email(
+            BoroEmailTemplateNames.ISSUANCE, operation.operator.legal_name, operation, user
```

Comment:
We need to remove the `user` from here! We don't need to send this email to the `cas_director`.